### ISSUE TITLE
Remove references to 'focusable'

### ISSUE THREAD

**Issue by jessegreenberg** [phetsims/sun#225]
The 'focusable' property of scenery/Node.js is now deprecated.  User interface components in sun will be implicitly added to the navigation order in the parallel DOM without ```focusable: true```.

There are six references to ```focusable: true``` in sun files, and these should be removed.

**Comment by jessegreenberg** [comment]
Done, closing.
